src/integrationTest/java/uk/gov/hmcts/darts/usermanagement/controller/PatchSecurityGroupIntTest.java

```diff
@@ -0,0 +1,346 @@
+                           "name": "<name>",
+                           "display_name": "<display_name>",
+                           "description": "<description>"
+                         }
+                           """;
+        String newName = "security group name" + UUID.randomUUID();
+        patchContent = patchContent.replace("<name>", newName);
+        patchContent = patchContent.replace("<display_name>", NEW_DISPLAY_NAME);
+        patchContent = patchContent.replace("<description>", NEW_DESCRIPTION);
+
+        String name = "security group name" + UUID.randomUUID();
+        Integer id = createSecurityGroup(name);
+
+        MockHttpServletRequestBuilder patchRequest = buildPatchRequest(id)
+            .content(patchContent);
+
+        mockMvc.perform(patchRequest)
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("$.id").value(id))
+            .andExpect(jsonPath("$.name").value(newName))
+            .andExpect(jsonPath("$.display_name").value(NEW_DISPLAY_NAME))
+            .andExpect(jsonPath("$.description").value(NEW_DESCRIPTION))
+            .andExpect(jsonPath("$.global_access").value(false))
+            .andExpect(jsonPath("$.security_role_id").isNumber())
+            .andExpect(jsonPath("$.courthouse_ids").isEmpty())
+            .andExpect(jsonPath("$.user_ids").isEmpty())
+            .andReturn();
+    }
+
+    @Test
+    void patchSecurityGroupShouldSucceedWhenProvidedWithCourthouseIds() throws Exception {
+        superAdminUserStub.givenUserIsAuthorised(userIdentity);
+
+        String name = "security group name" + UUID.randomUUID();
+        Integer id = createSecurityGroup(name);
+
+        var courthouseEntity1 = dartsDatabase.createCourthouseUnlessExists(TEST_COURTHOUSE_NAME_1);
+        var courthouseEntity2 = dartsDatabase.createCourthouseUnlessExists(TEST_COURTHOUSE_NAME_2);
+
+        String patchContent = String.format("{\"courthouse_ids\": [%s, %s]}", courthouseEntity1.getId(), courthouseEntity2.getId());
+
+        MockHttpServletRequestBuilder patchRequest = buildPatchRequest(id)
+            .content(patchContent);
+
+        mockMvc.perform(patchRequest)
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("$.id").value(id))
+            .andExpect(jsonPath("$.name").value(name))
+            .andExpect(jsonPath("$.display_name").value(ORIGINAL_DISPLAY_NAME))
+            .andExpect(jsonPath("$.description").value(ORIGINAL_DESCRIPTION))
+            .andExpect(jsonPath("$.global_access").value(false))
+            .andExpect(jsonPath("$.security_role_id").isNumber())
+            .andExpect(jsonPath("$.courthouse_ids[0]").value(courthouseEntity1.getId()))
+            .andExpect(jsonPath("$.courthouse_ids[1]").value(courthouseEntity2.getId()))
+            .andExpect(jsonPath("$.user_ids").isEmpty())
+            .andReturn();
+    }
+
+    @Test
+    void patchSecurityGroupShouldSucceedWhenProvidedWithUserIds() throws Exception {
```

Comment:
When I try the PATCH containing a user_id when running locally the change doesn't appear to "stick". I.e I'd expect a record to be created in `security_group_user_account_ae`, but it doesn't appear to be, despite the ids appearing in the repsonse. Maybe I'm missing it, but can you check that scenario when running the full app locally against a db? 

Also would be worth checking the db state is as expected here in the integration test.